data_harvester:
  ros__parameters:
    runtime_min: 0.2
    map_name: 'harvested_map'